jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -51,413 +53,605 @@
     - kubernetes-e2e-aws:
         job-name: ci-kubernetes-e2e-aws
         frequency: 'H/5 * * * *' # At least every 30m
+        timeout: 140
```

Comment:
For jobs you aren't changing please leave the timeout at 0.

Also given the enormous number of jobs you're changing, will you send me a separate PR that adds `--json='{json}' --timeout='{timeout}'` to the template, and then sets these values to zero to all jobs?